dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -1410,11 +1410,11 @@ public void findAllSubCommunitiesGrantAccessAdminsTest() throws Exception {
                 .withAdminGroup(child2Admin)
                 .build();
 
-        Community ommunityChild1Child1 = CommunityBuilder.createSubCommunity(context, communityChild1)
+        Community communityChild1Child1 = CommunityBuilder.createSubCommunity(context, communityChild1)
                 .withName("Sub1 Community 1")
                 .build();
 
-        Community сommunityChild2Child1 = CommunityBuilder.createSubCommunity(context, communityChild2)
```

Comment:
EP said that this line contained a Unicode character which should not be in code.  I could never find it so I retyped the line as I saw it, and deleted the original.